ui/pages/confirmations/utils/confirm.ts

```diff
@@ -46,7 +46,7 @@ export const parseSanitizeTypedDataMessage = (dataToParse: string) => {
 };
 
 export const isSIWESignatureRequest = (request: SignatureRequestType) =>
-  request.msgParams?.siwe?.isSIWEMessage;
+  request?.msgParams?.siwe?.isSIWEMessage;
```

Comment:
Minor, but should we wrap this with a `Boolean()` so it always returns a `boolean` rather than `undefined`?

Reply:
great call! Updated f779176f54f3529d2c1ee1d94e53d3be2c089cb1